docs/tutorials/01_neural_networks.ipynb

```diff
@@ -434,26 +549,42 @@
        "         [ 0.3924887 ,  0.0382929 ,  0.29041568, -0.0128533 ]]]))"
       ]
      },
-     "execution_count": 19,
+     "execution_count": 16,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# QNN backward pass, returns a tuple of matrices, None for the gradients with respect to input data.\n",
     "qnn4.backward(input4, weights4)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "66117e82",
+   "metadata": {},
+   "source": [
+    "## 5. Conclusion\n",
+    "In this tutorial, we learn about the two neural networks provided by Qiskit Machine Learning, namely `EstimatorQNN` and `SamplerQNN`, which extend the based `NeuralNetwork` class.\n",
+    "\n",
+    "For EstimatorQNN we implemented the following steps:\n",
+    "1. Construct an EstimatorQNN using a circuit, observable, input and weights\n",
+    "2. Perform Forward Pass and batched Forward Pass\n",
+    "3. Perform Backward Pass and batched Backward Pass\n",
+    "4. Combine multiple observables for more complex workflows\n",
+    "\n",
+    "Likewise for `SamplerQNN`, we defined the neural network using a circuit and iterated through forward and backward propagation."
```

Comment:
I think the part for `SamplerQNN` should be similar to what is written about `EstimatorQNN`.

Reply:
Done